Tw1

MenteeVelga51 opened the pull request:
Please check and add Login form and Registration success page

Commit:
TW1 - Create Footer component - Fixed hydration issue

MenteeVelga51 committed:
Login form

Login form with 4 errors I don't see where

MenteeVelga51 committed:
Login and Registration success page

Login form and Registration success page finalized

MenteeVelga51 committed:
RegistrationSuccessPage

sliceem88 commented on src/components/Loginform/Loginform.jsx:
```diff
@@ -0,0 +1,16 @@
+import './LoginForm.scss'
```
I don't see this LoginForm.scss file any where

sliceem88 replied:
Maybe you forgot to push it ??

MenteeVelga51 replied:
Will create the file

sliceem88 commented on src/components/RegistrationSuccessPage/RegistrationSuccessPage.jsx:
```diff
@@ -0,0 +1,11 @@
+
+    "use client";
+    import   
```
You have key word `import` , but I don't see that you are importing something

sliceem88 commented on src/components/RegistrationSuccessPage/RegistrationSuccessPage.jsx:
```diff
@@ -0,0 +1,11 @@
+    import   
+    function Registrationsuccesspage () {
+      return <head>Registration success page</head>
+      function buildURL(item)
```
Syntaxis is wrong. Please see React documentation how it should look like, Or check your code from `LoginForm` component.

1. return should contain one element/tag
2. tag/element can have a lot of children
3. `function buildURL(item)` shoul be out of return statement
  You can check this example of component:
  
```
  import React from 'react';
export const renderButton = (props) => {
    const handleClick = () => {
        console.log('handleClick() props data from App: ' + props.dataFromApp)

    }
    return (
        <button onClick={handleClick}>Click</button>
    )
}
```

sliceem88 commented on src/components/Loginform/Loginform.jsx:
```diff
@@ -0,0 +1,16 @@
+      <input type="text" placeholder="Enter Username" id="username" required />
+      <label for="password"><b>Parole</b></label>
+      <input type="password" placeholder="Enter Password" id="password" required />
+      <button onclick="login()">Pierakstīties</button>
```
onclick event name is wrong, it should be `onClick`  - it is important
login() function itself should be before `return` statement

sliceem88 commented:
Also I don't see where your component is used

MenteeVelga51 committed:
Login form updates to comments

Here are some changes I made to the Loginform. Less than in the TW-21-Login-form though
